demo-output.yaml

```diff
@@ -166,26 +166,6 @@
         variables:
           file: file:///analyzer-lsp/examples/golang/main.go
       extras: []
-    golang-gomod-dependencies:
```

Comment:
Don't commit changes to this file, we don't want the output to be different so if this file is changed it means the changeover isn't complete

Reply:
my bad, the file was included accidentally while i was commiting